src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private ColorSupplier colorSupplier;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(100);
+        Color randomColor = colorSupplier.getRandomColor();
+        return new Ball(randomColor.toString(), randomNumber);
```

Comment:
[checklist issue](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)

Reply:
not fixed
![image](https://github.com/mate-academy/jv-lottery/assets/49767049/1890d353-7fac-496a-926b-b209fb6d700b)